pkgs/default.nix

```diff
@@ -1,6 +1,5 @@
 { pkgs ? import <nixpkgs> {} }:
 {
-  nodeinfo = pkgs.callPackage ./nodeinfo { };
```

Comment:
why make nodeinfo a module? I can see some benefits like being able to easily access nix variables but this does not seem to happen in the module.

Reply:
Oh I missed that the operator user is used in the script! Would be better if the operator-name was an option in the secure-node.nix

Reply:
Resolved in ca8476a